src/D2L.Bmx/Program.cs

```diff
@@ -215,6 +215,13 @@
 		next
 	) => {
 		await UpdateChecker.CheckForUpdatesAsync( configProvider.GetConfiguration() );
```

Comment:
Note: This check creates .bmx folder if it doesn't exist already so we can remove the other folder checks